masschallenge/django-accelerator
# [AC-4325] Add default values for ApplicationQuestion text box fields

[JIRA](https://masschallenge.atlassian.net/browse/AC-4325)

These fields are not supposed to be blank left blank, so i added defaults

I chose `1` for text_box_lines and `500` for text_limit

I chose these value because they seem like good defaults, that in most cases, the person creating the question will not have to change. To the best of my knowledge, having nonzero values in these fields has no effect on creating non-text questions

To test:
- check out this branch
- go to http://localhost:8181/admin/mc/applicationquestion/add/
- default values should exist in the "Text Box Lines" and "Text Limit" fields
